build.sbt

```diff
@@ -39,7 +23,8 @@ val common = Seq(
 )
 
 lazy val root = (project in file("."))
-  .settings(common :+ (name := "effect-util"))
+  .settings(common :+ (name := "effect-utilities"))
```

Comment:
oh also I have a feeling we could / should consider renaming this repo `effect-utilities` - I slightly regret the non plural and shortened `util`